scripts/lib/api/updateLinks.test.ts

```diff
@@ -51,7 +52,37 @@ describe("updateLinks", () => {
       },
     ];
 
-    await updateLinks(data);
+    const objectsInvEntries = [
+      "stubs/qiskit.algorithms.Eigensolver.html#$",
+      "stubs/qiskit.algorithms.Eigensolver.html#$",
+      "stubs/qiskit.algorithms.EigensolverResult.html#$",
+      "stubs/qiskit.algorithms.EstimationProblem.html#$",
+      "stubs/qiskit.algorithms.EvolutionProblem.html#$",
+      "stubs/qiskit.algorithms.EvolutionProblem.html#$",
+      "stubs/qiskit.algorithms.FasterAmplitudeEstimationResult.html#$",
+      "stubs/qiskit.algorithms.FasterAmplitudeEstimationResult.html#$",
+      "stubs/qiskit_ibm_runtime.QiskitRuntimeService.html",
+      "stubs/qiskit_ibm_runtime.RuntimeJob.html#qiskit_ibm_runtime.RuntimeJob.submit",
+      "stubs/qiskit_ibm_runtime.RuntimeEncoder.html#qiskit_ibm_runtime.RuntimeEncoder",
+      "stubs/qiskit_ibm_runtime.options.Options.html#options",
+      "tutorials/qaoa_with_primitives.html",
+      "tutorials/vqe_with_estimator.html#Step-1:-Map-classical-inputs-to-a-quantum-problem",
+      "qiskit.algorithms.gradients.LinCombEstimatorGradient#SUPPORTED_GATES",
```

Comment:
Is this missing a `.html` ending?

Reply:
Actually the other way round, I've since had `ObjectsInv.ts` strip the `.html` but forgot to update the tests; fixed in 5078141560793664c97b13e5f95e65da2e6e91d1.